tests/plugins/pastefromwordimage/manual/pastewordwithimage.md

```diff
@@ -13,6 +13,9 @@
   2. [Online_and_offline_image.docx](../generated/_fixtures/Online_and_offline_image/Online_and_offline_image.docx)
   3. [Shapes_and_online_and_offline_image.docx](../generated/_fixtures/Shapes_and_online_and_offline_image/Shapes_and_online_and_offline_image.docx)
   4. [Wrapped_image.docx](../generated/_fixtures/Wrapped_image/Wrapped_image.docx)
+  5. [Image_alternative_text.docx](../generated/_fixtures/Image_alternative_text/Image_alternative_text.docx)
+  6. [Image_reflection.docx](../generated/_fixtures/Image_reflection/Image_reflection.docx)
+  7. [Image_rotation.docx](../generated/_fixtures/Image_rotation/Image_rotation.docx)
 
 **Expected:** Images are pasted to the editor.<br>
 Note: Keep in mind that shapes (like last object in `Shapes_and_online_and_offline_image.docx`) will not be pasted.
```

Comment:
To be honest I cannot recall what was the status of handling shapes. Now I see that for shapes (e.g. in `Shapes_and_online_and_offline_image.docx`), `img` element with invalid path (pointing to some local/temporary resources) is inserted. Shouldn't it just be skipped? Same happens for canvas elements.

Reply:
I know that this PR is about images, but as a whole implementation of `v:shape` filter changed it influences shapes also.

Reply:
It's a good point. Honestly I don't know. I remain them untouched. I'll talk with @mlewand what to do with it.

Reply:
Ok so img which earlier appear after pasting shapes are now removed. More details how it works is below in main line comment.